option optimize_for = LITE_RUNTIME;

message HeartbeatAnswer
{
 required uint32 result = 1;
 required uint32 times = 2;
}
